src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
```

Comment:
At the same time we want to make sure users when they include `syscall.h` don't get a ton of random structs either. Can we make a header file within the graphics module that contains all structs exposed to the public?